src/main/webapp/js/navigation-loader.js

```diff
@@ -103,19 +103,16 @@ function scrollTransparency() {
   }
 
   let scrollStart = 0;
-  const landingPageDescription = $('.landing-page-description');
-  const offset = landingPageDescription.offset();
-  if (landingPageDescription.length) {
-    $(document).scroll(function changeTransparencyOnScroll() {
-      const navBar = document.getElementById('navigationBar');
-      scrollStart = $(this).scrollTop();
-      if (scrollStart > offset.top) {
-        navBar.className = 'navbar navbar-expand-lg d-flex fixed-top border-bottom shadow-sm navbar-light bg-light';
-      } else {
-        navBar.className = 'navbar navbar-expand-lg d-flex fixed-top navbar-dark transparent navBarTransparent';
-      }
-    });
-  }
+  const offset = 200;
```

Comment:
Should the offset be a fixed number?

Reply:
I was using this website as the inspiration: https://feras-creative-one-page-parallax.netlify.com/video.html. And I think it is ok.